recipes/ipython-memory-usage/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ipython-memory-usage" %}
+{% set version = "1.1" %}
+{% set sha256 = "1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ianozsvald/ipython_memory_usage.git
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+
+about:
+  home: https://github.com/ianozsvald/ipython_memory_usage
+  license: BSD
+  license_file: LICENSE
```

Comment:
The `license_file` won't be included in the tarball on pypi. You can manually package it:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and then raise an issue on the github repo asking that the license be included. Put the url to the issue as a comment in the `meta.yaml` file as a reminder to check the issue's status for future updates to the feedstock.